cmd/config.go

```diff
@@ -236,6 +236,22 @@ func getAccount() (*account, error) {
 		fmt.Printf("Checking the credentials of %q...", account.Key)
 		resp, err := client.GetWithContext(gContext, egoscale.Account{})
 		if err != nil {
+			if egoerr, ok := err.(*egoscale.ErrorResponse); ok && egoerr.ErrorCode == egoscale.ErrorCode(403) {
+				fmt.Println("This key is restricted, Please enter your account name.")
```

Comment:
This error message is confusing: the user doesn't necessary know what is the meaning of *restricted* and what is has to do with the fact that we're asking him/her to provide account information.

You should replace `Checking the credentials of %q...` (because we're actually doing more than just checking the credentials) with `Retrieving account information...`, then if the specified API key is indeed restricted and doesn't allow the CLI to do it then return a message ` failed. Please enter your account information:`, or simply ` done` if the account information has been successfully retrieved.

Reply:
I agree with you 👍